fdbserver/MoveKeys.actor.cpp

```diff
@@ -1868,6 +1875,16 @@ ACTOR static Future<Void> startMoveShards(Database occ,
 					dataMove.ranges.clear();
 					dataMove.ranges.push_back(KeyRangeRef(keys.begin, currentKeys.end));
 					dataMove.dest.insert(servers.begin(), servers.end());
+					dataMove.dcTeamIds = std::unordered_map<std::string, std::string>();
+					for (auto& [dc, serverIds] : dcServers) {
+						std::sort(serverIds.begin(), serverIds.end());
+						std::string teamId;
+						for (const auto& serverId : serverIds) {
+							teamId += serverId;
```

Comment:
Will this be a long string? 3 x UUID?  Will it be easy for human to read in log?  (I think you talked of hashing the team member ids together elsewhere?) I suppose it'll be nice having an id made of serverids because then you can decipher who makes up the team. If so, maybe add in a delimiter so human can decipher start/end?

Reply:
I'm taking the first part of the UUID. For replication factor of 3, the id length is 96. Using a concatenation is the easiest way to construct it. If in the future we need a larger team, we could tweak the id generation part.